### PR TITLE
(release-notes): updated release notes for 2020.02 release

### PR DESCRIPTION
**Pull request by rachael-phillips**
Please review the release notes for 2020.02

### CODE REVIEW

**Comment by rachael-phillips** [comment]
Hey @redallen when do we add the package name in?

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -4,6 +4,38 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.02 release notes (2020-02-18)
+Packages released:
+
+### Components
+- **Data list:** Enabled hidden/visible feature to update display ([#2691](https://github.com/patternfly/patternfly-next/pull/2691))
+- **Data list, compact example:** Added compact version of data list ([#2686](https://github.com/patternfly/patternfly-next/pull/2686))
```
Comment (redallen) [inline RELEASE-NOTES.md]: Maybe just put two bullet points under data list? `Data list, compact example` isn't a component.

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -4,6 +4,38 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.02 release notes (2020-02-18)
+Packages released:
+
```
Comment (redallen) [inline RELEASE-NOTES.md]: Do we need to include the package version here? If so, the latest on https://www.npmjs.com/package/@patternfly/patternfly is 2.63.0.

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -4,6 +4,38 @@ section: overview
...
+### Components
+- **Data list:** Enabled hidden/visible feature to update display ([#2691](https://github.com/patternfly/patternfly-next/pull/2691))
+- **Data list, compact example:** Added compact version of data list ([#2686](https://github.com/patternfly/patternfly-next/pull/2686))
+- **Data toolbar:** 
+  - Updated input type to search ([#2648](https://github.com/patternfly/patternfly-next/pull/2648))
+  - Made Data toolbar experimental ([#2611](https://github.com/patternfly/patternfly-next/pull/2611))
+- **File upload:** Added File upload component ([#2681](https://github.com/patternfly/patternfly-next/pull/2681))
```
Comment (mattnolting) [inline RELEASE-NOTES.md]: **Inline edit:** Fixes Firefox alignment bug
https://github.com/patternfly/patternfly-next/commit/6f56e1655bd31b2c77ebe04d0747ac9b114b5cd3

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -4,6 +4,38 @@ section: overview
...
+- **Master detail:** Added Master detail demo ([#2645](https://github.com/patternfly/patternfly-next/pull/2645))
```
Comment (mattnolting) [inline RELEASE-NOTES.md]:
```suggestion
- **Master detail:** Added master-detail demo ([#2645](https://github.com/patternfly/patternfly-next/pull/2645))
```

File: RELEASE-NOTES.md (hunk 1 above)
Comment (mattnolting) [inline RELEASE-NOTES.md]:
```suggestion
- **Data list, compact example:** Added compact version of data-list ([#2686](https://github.com/patternfly/patternfly-next/pull/2686))
```

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -4,6 +4,38 @@ section: overview
...
 
+## 2020.02 release notes (2020-02-18)
+Packages released:
+
+### Components
+- **Data list:** Enabled hidden/visible feature to update display ([#2691](https://github.com/patternfly/patternfly-next/pull/2691))
+- **Data list, compact example:** Added compact version of data list ([#2686](https://github.com/patternfly/patternfly-next/pull/2686))
+- **Data toolbar:** 
+  - Updated input type to search ([#2648](https://github.com/patternfly/patternfly-next/pull/2648))
+  - Made Data toolbar experimental ([#2611](https://github.com/patternfly/patternfly-next/pull/2611))
```
Comment (mattnolting) [inline RELEASE-NOTES.md]:
```suggestion
  - Made data-toolbar experimental ([#2611](https://github.com/patternfly/patternfly-next/pull/2611))
```

File: RELEASE-NOTES.md (hunk 3 above)
Comment (mattnolting) [inline RELEASE-NOTES.md]:
```suggestion
- **File upload:** Added file-upload component ([#2681](https://github.com/patternfly/patternfly-next/pull/2681))
```

Comment (rachael-phillips) [reply RELEASE-NOTES.md]: Yes, I'll add it

File: RELEASE-NOTES.md (hunk 4 above)
Comment (rachael-phillips) [reply RELEASE-NOTES.md]: I'm going to leave this alone until I hear back on what the name punctuation should be.

File: RELEASE-NOTES.md (hunk 1 above)
Comment (rachael-phillips) [reply RELEASE-NOTES.md]: We don't need the - here.

File: RELEASE-NOTES.md (hunk 2 above)
Comment (rachael-phillips) [reply RELEASE-NOTES.md]: Yes, thank you!

File: RELEASE-NOTES.md (hunk 1 above)
Comment (rachael-phillips) [reply RELEASE-NOTES.md]: Great point!

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -4,6 +4,40 @@ section: overview
 releaseNoteTOC: true
 ---
 
+## 2020.02 release notes (2020-02-18)
+Packages released: @patternfly/patternfly@v2.63.0
```
Comment (redallen) [inline RELEASE-NOTES.md]: Nit: add a dash to make it a bullet point like all previous release notes

**Pull request merged**